pom.xml

```diff
@@ -45,10 +45,11 @@
         <air.java.version>1.8.0-40</air.java.version>
 
         <dep.antlr.version>4.5.1</dep.antlr.version>
-        <dep.airlift.version>0.125</dep.airlift.version>
+        <dep.airlift.version>0.127-SNAPSHOT</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.19</dep.slice.version>
         <dep.aws-sdk.version>1.9.40</dep.aws-sdk.version>
+        <dep.tempto.version>1.7-SNAPSHOT</dep.tempto.version>
```

Comment:
`Airlift` is still `SNAPSHOT` anyway. Once `Airlift` is released we can release `Tempto` as well.